src/_includes/layouts/home.njk

```diff
@@ -2,4 +2,41 @@
 layout: layouts/base.njk
 templateClass: tmpl-home
 ---
-{{ content | safe }}
+
+<div>
+    <div class="deven-headtext">
+        <span>DEVEN</span>
+    </div>
+    <div class="abstract-geometry">
+        <div class="button-group">
+            <div class="pe-button">
+                <span class="abstract-text-span">product engineering</span>
+            </div>
+            <div class="stdtion-button">
+                <span class="abstract-text-span">standarization</span>
+            </div> 
+        </div>
+        <div class="ellipse">
+            <span>👩‍💻</span>
+        </div>
+        <div class="button-group1">
+            <div class="dev-square">
+                <span class="abstract-text-span">developer</span>
+            </div>
+            <div class="link-button">
+                <span class="abstract-text-span"><svg width="26" height="37" viewBox="0 0 26 37" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M0.0615453 24.3116L2.01155 22.3616L10.2115 30.5616L11.3115 32.1116L11.8115 31.9616L11.6115 30.0616L11.6115 0.961551L14.4115 0.961551L14.4115 30.0616L14.2115 31.9616L14.7115 32.1116L15.8115 30.5616L24.0115 22.3616L25.9615 24.3116L14.0115 36.2616L12.0115 36.2616L0.0615453 24.3116Z" fill="black"/>
+</svg>
+</span>
+            </div>
+            <div class="enable-circle">
+                <span class="abstract-text-span">enablement</span>
+            </div>
+        </div>
+        <div class="four-point-star">
```

Comment:
I noticed that the edges are getting rounded off. This is probably to do with the system font. This will also happen when using Space Grotesk. This is an alternative that I came up with:
`.four-point-star {
    position: absolute;
    z-index: 1;
    left: 43rem;
    top: -43rem;
    background-image: url(data:image/svg+xml,%3Csvg width='408' height='408' viewBox='0 0 408 408' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M204 0C204 0 185.455 80.3636 134.455 134.455C83.4545 188.545 0 204 0 204C0 204 83.4545 219.455 134.455 273.545C185.455 327.636 204 408 204 408C204 408 222.545 327.636 273.545 273.545C324.545 219.455 408 204 408 204C408 204 324.545 188.545 273.545 134.455C222.545 80.3636 204 0 204 0Z' fill='%23F1DE00'/%3E%3C/svg%3E);
    width: 400px;
    height: 400px;
    background-repeat: no-repeat;
    background-size: contain;
    display: flex;
    justify-content: center;
    align-items: center;
}
.four-point-star span {
    z-index: 2;
    font-family: "Space Grotesk";
    font-style: normal;
    font-weight: 700;
    font-size: 20px;
    line-height: 26px;
}
`
and remove the :before and :after pseudo elements. This should work. You might need to fine tune it a little to be more responsive or more like the Figma

Reply:
Before: <img width="355" alt="Screenshot 2023-04-19 at 15 46 34" src="https://user-images.githubusercontent.com/127498135/233095295-e58aeeb4-5934-4e3f-8e65-bc0270ed393e.png">
After: 
<img width="508" alt="Screenshot 2023-04-19 at 15 51 12" src="https://user-images.githubusercontent.com/127498135/233096390-3096677e-442d-4260-8805-a65dd4436c30.png">

Reply:
No it is not a system font issue. But issue with `content: "\2726";` itself. It is very hard geometry to design. Using svg is good if we want to come close to design. In your above approach svg is used. Earlier we did not had svg. And SVG are way more heavier than css.